react-accordion-component/src/index.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react"
+import ReactDOM from "react-dom"
+import {Accordion, AccordionR} from "./accordion"
+
+ReactDOM.render(
+  <Accordion />, document.querySelector("#root")
```

Comment:
Your content array should be moved to here and passed to your `Accordion` via props